src/main/java/org/jenkinsci/tracey/TraceyTrigger.java

```diff
@@ -66,7 +71,10 @@
     public void start(final Job<?,?> project, boolean newInstance) {
         super.start(project, newInstance);
 
-        broker = configureBroker(project, traceyHost, type, exchange);
+        broker = configureBroker(project, traceyHost);
```

Comment:
It would be great to rename traceyHost to traceyHostId in order to highlight that this is something related to global config and not host name